src/Components/Publishing/__stories__/Article.story.tsx

```diff
@@ -7,19 +7,33 @@ import {
   ShortStandardArticle,
   StandardArticle,
 } from "../Fixtures/Articles"
-import { RelatedCanvas, RelatedPanel } from "../Fixtures/Components"
+import { Display, RelatedCanvas, RelatedPanel } from "../Fixtures/Components"
 
-storiesOf("Publishing/Articles", module)
-  .add("Standard", () => {
+const story = storiesOf("Publishing/Articles", module).add("Standard", () => {
+  return (
+    <Article
+      article={StandardArticle}
+      relatedArticlesForPanel={RelatedPanel}
+      relatedArticlesForCanvas={RelatedCanvas}
+      emailSignupUrl="#"
+    />
+  )
+})
+;["overlay", "image", "video", "slideshow"].forEach(type => {
+  story.add(`Standard with ${type} ad`, () => {
```

Comment:
same idea here -- standard layouts render image or video

Reply:
This is for creating stories for each of the media types—kinda confusing b/c of the `type` field before so I renamed it to `mediaType`, but I think this makes sense.

![image](https://user-images.githubusercontent.com/555859/31559537-f5ca53ca-b01e-11e7-8481-22965be42e15.png)